addon/ng2/blueprints/ng2/files/package.json

```diff
@@ -2,6 +2,9 @@
   "name": "<%= htmlComponentName %>",
   "version": "0.0.0",
   "license": "Apache-2.0",
+  "scripts":{
+    "postinstall": "webdriver-manager update"
```

Comment:
I'm not entirely sure what you're proposing. `webdriver-manager update` will only actually download stuff if it finds it needs new files, so it already does this? Unless you mean something different by "no driver at all" - like looking in some shared location?

I know the slow download time is a pain point, so I'm happy to take ideas and figure out a better solution overall.